tests/test_data_sync_client.py

```diff
@@ -45,35 +42,37 @@ class TestClient:
     async def test_binary_data_capture_upload(self, service: MockDataSync):
         async with ChannelFor([service]) as channel:
             client = DataClient(channel, DATA_SERVICE_METADATA)
-            await client.binary_data_capture_upload(
+            response = await client.binary_data_capture_upload(
```

Comment:
[minor] to match the naming convention in the existing tests below

```suggestion
            file_id = await client.binary_data_capture_upload(
```

Reply:
updated for both tests